src/icalendar/prop.py

```diff
@@ -107,17 +108,12 @@ def dst(self, dt):
 class LocalTimezone(tzinfo):
     """Timezone of the machine where the code is running.
     """
+
     def utcoffset(self, dt):
-        if self._isdst(dt):
-            return DSTOFFSET
-        else:
-            return STDOFFSET
+        return self._isdst(dt) and DSTOFFSET or STDOFFSET
 
     def dst(self, dt):
-        if self._isdst(dt):
```

Comment:
If you want to change this:

(A if condition else B) is the way. Not and/or. Logical equivalent but confusing.

Reply:
Ok if that is the case. I shall revert back.

Reply:
Reverted to ```return A if condition else B``` constructs everywhere.